DockerfileRPI

```diff
@@ -0,0 +1,18 @@
+ARG ROS_DISTRO=jazzy
```

Comment:
Since we have two Docker files, let's create folder docker and move files here.
Please do not forget to use `git mv` command when you move already commited files to avoid conflicts.
Please use the following names for files:
- Dockerfile.sim - existing one
- Dockerfile.rpi - this file